presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -544,6 +562,25 @@ private void printDistributions(int indent, PlanNodeStats nodeStats)
         return ImmutableMap.of();
     }
 
+    private void printWindowNodeStats(int indent, WindowNodeStats stats)
+    {
+        output.append(indentString(indent));
+        output.append(format("Active Drivers: [ %d / %d ]", stats.getActiveDrivers(), stats.getTotalDrivers()));
```

Comment:
Right, but `EXPLAIN ANALYZE` is not intended for developers but rather average database user, therefore the driver information is not that useful. You can always get the driver data from the raw JSON.

Better would be to compute some kind of `parallelism` metric that would also account for the data skewness. Maybe it could be similar to the one used in GUI? Then such metric could be used by other plan nodes too.